README.md

```diff
@@ -153,12 +155,59 @@ EOF
 
 This registers the generator with Klyshko. Note that you have to specify each
 tuple type supported by the CRG and provide a recommended batch size for jobs
-that generate that type of tuples.
+that generate that type of tuples. Please consult the CRG documentation for more
+information.
 
 > **IMPORTANT**: In case a tuple type is supported by multiple generators no
 > tuples are generated for that tuple type to avoid potential inconsistencies
 > across VCPs.
 
+#### CRG Pod Template
+
+You can customize some aspects of the pod launched for a tuple generation task,
+i.e., the pod that hosts the container running the generator image. The
+following fields are customizable (in lexical order):
+
+| Aspect    | Description                                                                                        | Field(s)                                               |
+| --------- | -------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| Affinity  | Used to constrain on which nodes the generator pod can run (see [here][k8s-affinity] for details). | `spec.template.spec.affinity`                          |
+| Image     | The generator image to use (see [here][k8s-images] for details).                                   | `spec.template.spec.container.{image,imagePullPolicy}` |
+| Resources | How much resources the container needs (see [here][k8s-resource] for details).                     | `spec.template.spec.container.resources`               |
+
+Note that `spec.template.spec.container.image` is the only mandatory field. If a
+field is not provided the general default values for pods / containers are used
+(see links provided above).
+
+A fully customized sample generator pod template looks like the following:
+
+```yaml
+apiVersion: klyshko.carbnyestack.io/v1alpha1
+kind: TupleGenerator
+metadata:
+  name: mp-spdz-fake
+spec:
+  template:
+    affinity: # Only place pod on nodes running a Linux OS
```

Comment:
the table above defines the field for `Affinity` as  `spec.template.spec.affinity` , but in this example, `affinity` is a member of `template`, not `template.spec`

Reply:
Good catch! Fixed.